### PR TITLE
bug fix #30: no gaussian grids.

### LINKED ISSUE #30: bug with copernicus essential variables: we get reduced gaussian grid instead of regular
- many users would not know what to do with reduced gaussian grids so we better regrid data to regular latitude longitude.

### PR DESCRIPTION
**Pull request by annefou**
if data is retrieved in reduced gaussian grid, we regrid data to
regular latitude longitude so we can easily plot.

Should fix issue #30

### CODE REVIEW

File: tools/essential_climate_variables/essential_climate_variables.xml [hunk 1]
```diff
@@ -7,6 +7,7 @@
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
        HOME=`pwd`  &&
+       source \$COPERNICUS_CDSAPIRC_KEY_FILE 2> /dev/null | true &&
```
Comment (bgruening) [inline tools/essential_climate_variables/essential_climate_variables.xml]: I now do this: `cat \$COPERNICUS_CDSAPIRC_KEY_FILE > .cdsapirc | true &&
`

File: tools/essential_climate_variables/README.md [hunk 2]
```diff
@@ -10,8 +10,10 @@ To be able to retrieve data, you would need to register and get your [CDS API ke
 The CDS API Key needs to be:
   - located in a file called `.cdsapirc`in the HOME area 
     (as defined the `HOME` environment variable).
-  - or passed in an enviroment variable called `GALAXY_COPERNICUS_CDSAPIRC`. 
-    When passed in `GALAXY_COPERNICUS_CDSAPIRC`, make sure the key does not contain
-    the string `key: ` but the key itself only (starting with a number). The file
-    `.cdsapirc` will then be created and placed in the HOME area (using HOME
-    environment variable).
+  - or passed in an enviroment variable called `GALAXY_COPERNICUS_CDSAPIRC_KEY`.
```
Comment (bgruening) [inline tools/essential_climate_variables/README.md]: The file should look like that:

```
url: https://cds.climate.copernicus.eu/api/v2
key: 111:8789787-qwerz4e47-888-not-real
verify: 0

```

Comment (annefou) [reply tools/essential_climate_variables/README.md]: Godd idea: I will add some more information on how the key should look like.
Thanks.

File: tools/essential_climate_variables/essential_climate_variables.xml (hunk 1 above)
Comment (annefou) [reply tools/essential_climate_variables/essential_climate_variables.xml]: Thanks. I'll update the source code accordingly. I need to find out how to reduce the file size for the examples. I think I will add another parameter for the resolution so we can downgrade it at least for the test.

File: tools/essential_climate_variables/README.md (hunk 2 above)
Comment (bgruening) [reply tools/essential_climate_variables/README.md]: Cool thanks!

**Comment by annefou** [comment]
@bgruening I think I managed to fix this issue. It took longer than expected because Copernicus changed its interface too!

**Comment by bgruening** [comment]
Oo, how often do they do that? We should let them know that people are depending on this ... We have some nice collaboration with other data providers that do support us and at least inform us in advance.

File: tools/essential_climate_variables/essential_climate_variables.xml (hunk 1 above)
Comment (bgruening) [reply tools/essential_climate_variables/essential_climate_variables.xml]: ping ... this needs to be changed isn't it?

**Comment by bgruening** [comment]
Looks good to me.

**Comment by annefou** [comment]
> Oo, how often do they do that? We should let them know that people are depending on this ... We have some nice collaboration with other data providers that do support us and at least inform us in advance.

They are not supposed to do it very often. They mostly fixed bugs. I am hoping to be notified in the future (I asked to be notified).

**Pull request merged**